applications/MeshingApplication/custom_processes/mmg_process.cpp

```diff
@@ -677,6 +674,51 @@ void MmgProcess<TDim>::ExecuteRemeshing()
         }
     }
     
+    // TODO: Add OMP
+    // NOTE: We add the nodes from the elements and conditions to the respective submodelparts
+    const std::vector<std::string> sub_model_part_names = mrThisModelPart.GetSubModelPartNames();
+
+    for (auto sub_model_part_name : sub_model_part_names)
+    {
+        ModelPart& r_sub_model_part = mrThisModelPart.GetSubModelPart(sub_model_part_name);
+        
+        std::unordered_set<IndexType> node_ids;
+        
+        ConditionsArrayType& sub_conditions_array = r_sub_model_part.Conditions();
+        const SizeType sub_num_conditions = sub_conditions_array.end() - sub_conditions_array.begin();
+        
+//         #pragma omp parallel for 
```

Comment:
this openmp can not work, since it pushes back to node_ids.

I suggest just removing that (even as a comment)

Reply:
Ok